### ISSUE TITLE
Rely on X11 automatic configuration

### ISSUE THREAD

**Issue by Yoshi2889** [Antergos/Cnchi#669]
Nowadays X11 doesn't need to be told what driver to use for what GPU unless you're using a really awkward setup (which Cnchi isn't really able to handle anyway) and what screens map to where. I suggest we drop the configuration file generation and instead rely on X11 automatic configuration.
Arch ships with some default configuration which should be sufficient for most setups, according to their wiki page.

**Comment by karasu** [comment]
As far as I remember, we do include a keyboard setup (so Xorg loads the correct keyboard layout). Are you saying that this is no longer necessary?

**Comment by Yoshi2889** [comment]
It shouldn't be, no.

I was talking about the GPU configuration files btw, yesterday we had a
user who upgraded his GPU and it took us 3 hours or so to find out the
problem of the config file forcing an incompatible driver.

Op wo 16 nov. 2016 08:30 schreef karasu <notifications@github.com>:

**Comment by lots0logs** [comment]
Like @karasu , AFAIR we aren't generating any custom config for X related to the graphics driver. The only thing we do is for keyboard and touchpad (if applicable). We'll look into it though!

**Comment by Yoshi2889** [comment]
Could be I'm confusing things here, though has cnchi perhaps done this in the past?

**Comment by karasu** [comment]
the NVIDIA Xorg configuration tool creates a 20-nvidia.conf... maybe was this the problem. We'll check it anyways.

**Comment by karasu** [comment]
Well, I was wrong. Cnchi's hardware module creates a 20-nvidia.conf if privative drivers (nvidia) are selected.
```
def post_install(dest_dir):
        """ Post install commands """
        path = os.path.join(dest_dir, "etc/X11/xorg.conf.d/20-nvidia.conf")
        with open(path, 'w') as nvidia:
            nvidia.write('Section "Device"\n')
            nvidia.write('    Identifier "Nvidia Card"\n')
            nvidia.write('    Driver "nvidia"\n')
            nvidia.write('    VendorName "NVIDIA Corporation"\n')
            nvidia.write('    Option "NoLogo" "true"\n')
            nvidia.write('EndSection\n')
```
Should we stop doing this?

**Comment by lots0logs** [comment]
I dont think that its necessary anymore. I use the NVIDIA driver without any 20-nvidia.conf file on my system :thinking:

**Comment by karasu** [comment]
But it won't try then to use nouveau?

**Comment by karasu** [comment]
OK. I'll remove this and put xf86-video-nouveau in conflicts when installing nvidia.

**Comment by Yoshi2889** [comment]
hmm this was with Radeon upgrading to amdgpu so I think then the user ceeated it some other way. thanks for checking anyway
